javascript/javascript1/week2/eventday.js

```diff
@@ -0,0 +1,13 @@
+//creating a function
+function event(Day) {
```

Comment:
use `day` instead of `Day` because we usually in JavaScript give capital letters for the first character for classes which you will learn later.
- this name is too general. What about `getEventWeekday` instead